testing/live/DeeperDeepDungeonDex/manifest.toml

```diff
@@ -0,0 +1,5 @@
+[plugin]
```

Comment:
nonblocking;
seems like a few of the folders have a mismatching `.System` namespace - assuming a restructure that wasn't updated in-code?

Reply:
Nice catch, this has been fixed, thank you.